entity_api/src/coaching_relationship.rs

```diff
@@ -95,6 +134,43 @@ async fn by_organization(
     )
 }
 
+// A convenient combined struct that holds the results of looking up the Users associated
+// with the coach/coachee ids. This should be used as an implementation detail only.
+#[derive(FromQueryResult, Debug)]
+pub struct CoachingRelationshipWithUserNames {
+    pub id: Id,
+    pub coach_id: Id,
+    pub coachee_id: Id,
+    pub coach_first_name: String,
+    pub coach_last_name: String,
+    pub coachee_first_name: String,
+    pub coachee_last_name: String,
+    pub created_at: DateTimeWithTimeZone,
+    pub updated_at: DateTimeWithTimeZone,
+}
+
+// serialize the CoachingRelationshipUserWithNames struct so that it can be used in the API
+// and appears to be a coaching_relationship JSON object.
+impl Serialize for CoachingRelationshipWithUserNames {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: Serializer,
+    {
+        // 3 is the number of fields in the struct.
```

Comment:
Is 3 accurate? Does this comment serve a clarifying purpose, perhaps it was meant as a temporary comment?